engine/api/src/main/scala/pl/touk/nussknacker/engine/api/process/ExpressionConfig.scala

```diff
@@ -6,6 +6,7 @@ import pl.touk.nussknacker.engine.api.expression.ExpressionParser
 //TODO: how to make this config less spel-centric?, move globalImports and optimizeCompilation to spel configuration
 case class ExpressionConfig(globalProcessVariables: Map[String, WithCategories[AnyRef]],
                             globalImports: List[WithCategories[String]],
+                            allowedClasses: List[Class[_]],
```

Comment:
1. Can you use here default value like in list below? It is for compatibility reasons?
2. Also I think that we should rename it a bit because currently it looks like only allowed classes will be respected but in fact will be respected both classes extracted from component definition + allowed classes. Maybe it should be "additionalAllowedClasses" or "additionalClasses" or "addtionalTypes"? hmm

Reply:
Thanks, both changes improved.